pkg/remote/trans/netpollmux/server_handler.go

```diff
@@ -322,7 +327,8 @@ func (t *svrTransHandler) GracefulShutdown(ctx context.Context) error {
 	iv.SetSeqID(0)
 	ri := rpcinfo.NewRPCInfo(nil, nil, iv, nil, nil)
 	data := NewControlFrame()
-	msg := remote.NewMessage(data, t.svcInfo, ri, remote.Reply, remote.Server)
+	svcInfo := getRandomSvcInfo(t.svcInfoMap)
```

Comment:
Can I get a random svcInfo for graceful shutdown?

Reply:
Why don't you just use current svcInfo (eg. use svcInfo from Line 253)

Reply:
So you think it's better to add `svcInfo` field into `svrTransHandler` to store the target svcInfo for graceful shutdown?